### ISSUE TITLE
[Accessibility]: Code Insights - Add insight to dashboard (mobile)

### ISSUE THREAD

**Issue by unclejustin** [sourcegraph/sourcegraph#41765]
### Audit type

Mobile device navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33495

### Problem description

After adding an insight to the dashboard, the insight is appended to the bottom. On a mobile device, this happens offscreen. There is no "success" indication that the insight actually saved.

### Expected behavior

Either some "toast" alerts the user of the insight saved, or scroll to the bottom to see the added insight.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by vovakulikov** [comment]
I believe this is a valid problem for desktop client as well, in case if you have enough insight cards on the dashboard to add next one offscreen.
